Sources/WebSocketKit/WebSocket.swift

```diff
@@ -56,9 +88,11 @@ public final class WebSocket: Sendable {
         self._closeCode = .init(nil)
         self.frameSequence = .init(nil)
         self._pingInterval = .init(nil)
+        self.pmce = nil
     }
 
     @preconcurrency public func onText(_ callback: @Sendable @escaping (WebSocket, String) -> ()) {
+        print("\(self.onTextCallback._eventLoop)")
```

Comment:
Print statements must be removed

Reply:
PMCE RFC is somewhat obtuse. I'd be very surprised if it were perfectly implemented currently but it has worked against 4 different servers in my lab tests. These are all good suggestions and ive addressed most of them I think.